app/views/course/assessment/submission/submissions/_question.json.jbuilder

```diff
@@ -17,4 +17,14 @@ json.type case question.actable_type
             'Scribing'
           end
 
+# In order to be classified as a File Submission question
+# 	1. The question must be a Java question
+# 	2. There can be either no submission files or
+# 	3. The files must be .java files
+fileSubmission = answer.is_a?(Course::Assessment::Answer::Programming) &&
```

Comment:
That was actually stored within the package meta. Think it would be better to store it in a table rather than extract and parse from the package meta. What do you think?